pkg/handler/triggerhandler.go

```diff
@@ -812,8 +812,8 @@ func newStreamingHandler(writer http.ResponseWriter, sub pubsub.EventSubscriber)
 // TODO streamingHandler's methods should be merged into StreamingHandler as
 // unexported methods.
 func (sh *streamingHandler) handle(ctx context.Context, triggerID string) {
-	logger, _ := logger.GetZapLogger(ctx)
-	logger = logger.With(zap.String("triggerID", triggerID))
+	logger, _ := logx.GetZapLogger(ctx)
+	logger.Info("StreamingHandler", zap.String("triggerID", triggerID))
```

Comment:
Any reason to log here? Also, this change removes the `triggerID` param from future logs in this method

Reply:
This needs @donch1989 's input for the reason why logging here. I suspect it was some debug-purpose logging. It instantiated a new logger with the `triggerID` field by default but there is actually no further logs down the road.